staging/src/k8s.io/api/flowcontrol/v1beta1/BUILD

```diff
@@ -0,0 +1,36 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library")
+
+go_library(
+    name = "go_default_library",
+    srcs = [
+        "doc.go",
+        "generated.pb.go",
+        "register.go",
+        "types.go",
+        "zz_generated.deepcopy.go",
+        "zz_generated.prerelease-lifecycle.go",
```

Comment:
I believe this is missing `types_swagger_doc_generated.go`

Reply:
good catch!